__tests__/components/viewers/__snapshots__/stop-viewer.js.snap

```diff
@@ -606,56 +607,46 @@ exports[`components > viewers > stop viewer should render countdown times after
                       </IconWithText>
                     </button>
                   </div>
-                  <FromToLocationPicker
-                    label={true}
-                    onFromClick={[Function]}
-                    onToClick={[Function]}
+                  <span
+                    role="group"
                   >
-                    <strong>
-                      <FormattedMessage
-                        description="Label to prompt the user to plan a trip"
-                        id="otpUi.FromToLocationPicker.planATrip"
-                      >
-                        otpUi.FromToLocationPicker.planATrip
-                      </FormattedMessage>
-                    </strong>
-                    <styled__FromToPickerSpan>
-                      <span
-                        className="styled__FromToPickerSpan-sc-vb4790-1 dWHacc"
-                      >
-                        <styled__LocationPickerSpan>
-                          <span
-                            className="styled__LocationPickerSpan-sc-vb4790-0 fSBBPp"
-                          >
-                            <LocationIcon
-                              size="0.9em"
-                              type="from"
+                    <FromToLocationPicker
+                      label={true}
+                      onFromClick={[Function]}
+                      onToClick={[Function]}
+                    >
+                      <strong>
+                        <FormattedMessage
+                          description="Label to prompt the user to plan a trip"
+                          id="otpUi.FromToLocationPicker.planATrip"
+                        >
+                          otpUi.FromToLocationPicker.planATrip
+                        </FormattedMessage>
+                      </strong>
+                      <styled__FromToPickerSpan>
+                        <span
+                          className="styled__FromToPickerSpan-sc-vb4790-1 dWHacc"
+                        >
+                          <styled__LocationPickerSpan>
+                            <span
+                              className="styled__LocationPickerSpan-sc-vb4790-0 fSBBPp"
                             >
-                              <styled__FromIcon
-                                className=""
+                              <LocationIcon
                                 size="0.9em"
-                                title="From Location Icon"
+                                type="from"
                               >
-                                <DotCircle
-                                  className="styled__FromIcon-n5xcvc-0 bnFcGf"
+                                <styled__FromIcon
+                                  className=""
```

Comment:
Why the empty `className`s for `styled_FromIcon`/`styled_ToIcon`?

Reply:
I'm not sure! These files are auto-generated so who knows why it decided to do that haha